examples/echo.rs

```diff
@@ -1,26 +1,33 @@
 extern crate glitch_in_the_matrix as gm;
 extern crate futures;
 extern crate tokio_core;
+extern crate rpassword;
 
 use futures::{Future, Stream};
 use tokio_core::reactor::Core;
 use gm::{MatrixClient, MatrixFuture};
-use gm::types::{Content, Message};
+// use gm::types::{EventTypes, Content, Message};
+use gm::types::messages::{Message};
+use gm::types::content::{Content};
+use gm::types::events::{EventTypes};
+use rpassword::prompt_password_stdout;
 use std::env;
 
+
 fn main() {
-    let mut core = Core::new().unwrap();
-    let hdl = core.handle();
-    let mut args = env::args().skip(1).collect::<Vec<_>>();
-    if args.len() != 3 {
-        println!("Usage: cargo run --example echo -- SERVER USERNAME PASSWORD");
+    let args = env::args().skip(1).collect::<Vec<_>>();
+    if args.len() != 2 {
+        println!("Usage: cargo run --example echo -- SERVER USERNAME");
         return;
     }
-    let (server, username, password) = (&args[0], &args[1], &args[2]);
-    println!("server: {}\nusername: {}\npassword: {}", server, username, password);
+    let (server, username) = (&args[0], &args[1]);
+    println!("Type password for the bot (characters won't show up as you type them)");
+    let password = &prompt_password_stdout("password:").unwrap();
+    let mut core = Core::new().unwrap();
+    let hdl = core.handle();
     let mut mx = core.run(MatrixClient::login(username, password, server, &hdl)).unwrap();
-    println!("[+] Connected to Matrix.");
-    let mut ss = mx.get_sync_stream();
+    println!("[+] Connected to {} as {}",server,username);
```

Comment:
nit: ideally format this with spaces like `println!("[+] Connected to {} as {}", server, username);`